elide-async/src/main/java/com/yahoo/elide/async/service/DefaultAsyncQueryDAO.java

```diff
@@ -127,8 +149,38 @@ public AsyncQueryResult setAsyncQueryAndResult(Integer status, String responseBo
         return queryResultObj;
     }
 
-    @Override
-    public Object executeInTransaction(DataStore dataStore, Transactional action) {
+    @SuppressWarnings("unchecked")
+    public Collection<AsyncQuery> loadQueries(String filterExpression) {
+        EntityDictionary dictionary = elide.getElideSettings().getDictionary();
+        RSQLFilterDialect filterParser = new RSQLFilterDialect(dictionary);
+
+        Collection<AsyncQuery> loaded = (Collection<AsyncQuery>) executeInTransaction(dataStore, (tx, scope) -> {
+            try {
+                FilterExpression filter = filterParser.parseFilterExpression(filterExpression, AsyncQuery.class, false);
+
+                EntityProjection asyncQueryCollection = EntityProjection.builder()
+                        .type(AsyncQuery.class)
+                        .filterExpression(filter)
+                        .build();
+
+                Iterable<Object> loadedObj = tx.loadObjects(asyncQueryCollection, scope);
+                return loadedObj;
+            } catch (Exception e) {
```

Comment:
I wonder if this will swallow exceptions that would have been caught and handled by the outer transaction function.  Would it be better to let that function handle the exceptions?